python/openvino_tokenizers/convert_tokenizer.py

```diff
@@ -10,14 +10,55 @@
 from openvino.runtime.exceptions import OVTypeError
 
 from openvino_tokenizers.constants import UTF8ReplaceMode
-from openvino_tokenizers.utils import change_inputs_type, change_outputs_type, update_rt_info
-
+from openvino_tokenizers.utils import (
+    change_inputs_type,
+    change_outputs_type,
+    update_rt_info,
+    TokenzierConversionParams,
+)
 
 logger = logging.getLogger(__name__)
 
 
+def capture_arg(func):
+    from functools import wraps
```

Comment:
Move imports from the standard library to the top.

Reply:
done